dojson/utils.py

```diff
@@ -96,8 +96,12 @@ def wrapper(self, key, values, **kwargs):
 
 def force_list(data):
     """Wrap data in list."""
-    if data is not None and not isinstance(data, (list, tuple, set)):
-        return (data,)
+    if data is None:
+        return []
```

Comment:
@egabancho "There are only two hard things in Computer Science: cache invalidation and naming things". Yes, a docstring update would be good.